src/js/components/Brick.js

```diff
@@ -0,0 +1,47 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import Anchor from './Anchor';
+
+const CLASS_ROOT = "brick";
+
+export default class Brick extends Component {
+  render() {
+    var classes = [CLASS_ROOT];
+    classes.push('brick--' + this.props.width + '-' + this.props.height);
```

Comment:
can we use CLASS_ROOT and string templates here?

See this file for reference: https://github.com/grommet/grommet/blob/master/src/js/components/Distribution.js#L221.

As we moved to ES6 would you mind replacing `var` by `let`?